### DISCUSSION TITLE
Indicate what Field shared tables map to

### DISCUSSION THREAD

**Discussion by tlammer** [specify/specify7#5548] [Feature Requests]
For example Agent Last Name looks like this: 
![image](https://user-images.githubusercontent.com/12532430/131395716-eca8ecf4-caa3-44b0-89a4-76d7ee6fc44d.png)
I have agent last name in my mapping 4 times.  There should be some indication which field that agent last name connects to.

**Comment by maxpatiiuk** [comment]
Currently, it shows the icon of the table the field belongs to.
We could create an exception for Agent fields and use a parent table icon instead, if you are saying that would make it less confusing.
Though, when the "Uplaod Results" sidebar is opened, the entry for a total number of Agent records created would still be shown separately (alongside the number of records for the parent table, e.g, Accession Agents)


As an alternative, we could show the full mapping path when hovering over the table icon
